code/__HELPERS/lists.dm

```diff
@@ -261,6 +261,41 @@
 
 	return L[L.len]
 
+/// Takes a weighted list (see above) and expands it into raw entries
+/// This eats more memory, but saves time when actually picking from it
+/proc/expand_weights(list/list_to_pick)
```

Comment:
This is slightly undescriptive to me -- I had to read the code to figure out what the heck you meant.
I'd suggest adding an example.
```dm
// ex: list("a"=1, "b"=2) => list("a", "b", "b")
// ex: list("a"=2, "b"=4) => list("a", "b", "b") // because of a gcf of 2
```

Reply:
Added an example.